packages/amplify-graphql-conversation-transformer/src/__tests__/schemas/conversation-route-model-query-tool-with-relationships.graphql

```diff
@@ -34,7 +34,7 @@ type Mutation {
     systemPrompt: "You are a helpful chatbot. Answer questions to the best of your ability.",
     tools: [
       {
-        name: "list_customers",
+        name: "random_name",
```

Comment:
The name of this tool was changed because it was hiding the fact that we were (incorrectly) using the tool name to determine if a tool definition is a model list query tool. After changing the name, the test fails with the current implementation and works as expected with the changes introduced in this PR.